addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4291,21 +4292,7 @@ export class OdooEditor extends EventTarget {
         const node = ev.target;
         // handle checkbox lists
         if (node.tagName == 'LI' && getListMode(node.parentElement) == 'CL') {
-            const beforStyle = window.getComputedStyle(node, ':before');
-            const style1 = {
-                left: parseInt(beforStyle.getPropertyValue('left'), 10),
-                top: parseInt(beforStyle.getPropertyValue('top'), 10),
-            }
-            style1.right = style1.left + parseInt(beforStyle.getPropertyValue('width'), 10);
-            style1.bottom = style1.top + parseInt(beforStyle.getPropertyValue('height'), 10);
-
-            const isMouseInsideCheckboxBox =
-                ev.offsetX >= style1.left &&
-                ev.offsetX <= style1.right &&
-                ev.offsetY >= style1.top &&
-                ev.offsetY <= style1.bottom;
-
-            if (isMouseInsideCheckboxBox) {
+            if (isMouseInsideCheckbox(ev)) {
```

Comment:
```suggestion
        if (node.tagName == 'LI' && getListMode(node.parentElement) == 'CL' && isMouseInsideCheckbox(ev)) {
```
then outdent the contents.